spec/logr_spec.rb

```diff
@@ -0,0 +1,179 @@
+require 'spec_helper'
+require 'logger'
+require 'json'
+
+describe Logr do
+  context 'default DEBUG log level' do
+    let(:test_output) { StringIO.new }
+    let(:logger) { Logr::Logger.new('test_logger', Logger::DEBUG, test_output) }
```

Comment:
I would recommend keyword arguments over positional arguments so the intention is clear:

```ruby
let(:logger) { Logr::Logger.new('test_logger', Logger::WARN, test_output) }
```

vs

```ruby
let(:logger) { Logr::Logger.new('test_logger', level: Logger::WARN, log_device: test_output) }
```

The last argument is not really clear otherwise.